Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/db/DBRequestClient.java

```diff
@@ -30,7 +32,20 @@ public void init() {
     
     @Override
     public void store(ReviewRequest reviewRequest) {
-       
+        List<ReviewRequest> isReviewRequestPresent = client.query(new Predicate<ReviewRequest>() {
+            private static final long serialVersionUID = 6495914730735826451L;
+
+            @Override
+            public boolean match(ReviewRequest review) {
+                return review.getConceptId().equals(reviewRequest.getConceptId());
+            }
+            
+        });
+        //Deleting the reviewRequest if already present and Adding it back with updated Request data
+        if(isReviewRequestPresent.size() !=0) {
+            client.delete(isReviewRequestPresent.get(0));
+        }
```

Comment:
This scenario will occur if a specific concept is requested for review for the second time.

In this case, there will be data in all the fields (resolving comment, name, resolver etc.). So Instead of nullifying each field, I deleted and added a new one.

Reply:
don't delete existing reviews. we want to keep a history of everything that has happened.